src/LogFileReader.cpp

```diff
@@ -48,11 +48,24 @@ bool LogFileReader::readAll(const QString& fileName)
 bool LogFileReader::parseLine(const QString& line, BatteryData& batteryData) const
 {
     // TODO implement this first
+    QStringList splitLine = line.split(", ");
+      if(splitLine.length() != 3)
+        {
+           return false;
+        }
+
+     QString time = splitLine.at(0);
+     batteryData.time = QTime::fromString(time, "hh:mm:ss.zzz");
```

Comment:
This file is very clean, well done. I would note that in general writing strings in non print functions, as well as straight numbers in functions calls is poor structure. Normally, you would either derive such a value from somewhere else, or you would give it a descriptive name as a constant variable somewhere. 

In this case, the `"hh:mm:ss.zzz"` constant, since it can't be derived elsewhere, it should made as a constant. Normally how we do constants in C++ on the solar car team is adding a namespace at the top. So the code would be something like:
```c++
namespace 
{
    SOME_DESCRIPTIVE_NAME = "hh:mm:ss.zzz";
}
```